src/foam/dao/RequestResponseClientDAO.js

```diff
@@ -132,9 +132,10 @@ Suitable for usage against backends that don't support listen(), such as plain H
           }
           return superMethod(x, obj);
         }
-        
-        var result = obj.normalizeObj();
-        
+        if (obj && obj.normalizeObj && typeof obj.normalizeObj === 'function') {
+          var result = obj.normalizeObj();
```

Comment:
What if result isn't a promise? Just a regular object, then we won't send it instead?